components/List/list.css

```diff
@@ -0,0 +1,75 @@
+
+:root {
+  --list-background: rgb(245, 245, 245);
+  --list-border-color: #ff7474;
+  --list-item-color: rgb(88, 88, 88);
```

Comment:
if you are following RGB then use it everywhere or use hax only try not to mix up both.

Reply:
Okay